install/kubernetes/helm/istio-init/files/crd-10.yaml

```diff
@@ -63,6 +77,19 @@ spec:
     - networking-istio-io
   scope: Namespaced
   version: v1alpha3
+  additionalPrinterColumns:
+  - JSONPath: .spec.hosts
+    description: The hosts associated with the ServiceEntry
+    name: Hosts
+    type: string
+  - JSONPath: .spec.location
+    description: Whether the service should be considered external to the mesh or part of the mesh (MESH_EXTERNAL or MESH_INTERNAL)
```

Comment:
I would leave off the `should be considered` part, i.e.:

Whether the service is external to the mesh or part of ...

Reply:
@frankbu I will make this change.  Note that all the "description" text I use comes from istio.github.io/content/docs/reference/config/istio.networking.v1alpha3/index.html so if you have concerns about this text you should register them there as well.